Extension in ~/.k8slens/extensions blocks bundled extensions from loading

**Describe the bug**
even with newest lens version I still don't see the pod log- and terminal button.

**To Reproduce**
Install lense (no matter if "fresh" or with old versions and update) and no buttons there.

**Expected behavior**
I would expect to see the buttons ;-) 

**Screenshots**
![image](https://user-images.githubusercontent.com/63774617/126616261-b03e1857-1d9d-429d-8091-80cc5ab85ed8.png)


**Environment (please complete the following information):**
- Lens Version: 5.1.2-latest.20210719.1
![image](https://user-images.githubusercontent.com/63774617/126616360-03d53ac5-f697-499e-952f-32950fba8554.png)

- OS: Windows 10 Version 1909 (Build 18363.1645)
- Installation method (e.g. snap or AppImage in Linux): setup.exe

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
C:\Users\<myuser>\AppData\Local\Programs\Lens>lens.exe

C:\Users\<myuser>\AppData\Local\Programs\Lens>
info: ­ƒôƒ Setting Lens as protocol client for lens://
info: ­ƒôƒ Protocol client register succeeded Ô£à
info: ­ƒÜÇ Starting Lens from "C:\Users\<myuser>\AppData\Roaming\Lens"
info: ­ƒÉÜ Syncing shell environment
info: ­ƒÆ¥ Loading stores
Running 2.1.0-beta.4 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-user-store.json
Running 5.0.0-alpha.3 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-user-store.json
Running 5.0.3-beta.1 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-user-store.json
debug: UserStore.fromStore() {"lastSeenAppVersion":"0.0.0"}
info: [STORE]: LOADED from C:\Users\<myuser>\AppData\Roaming\Lens\lens-user-store.json
Running 2.0.0-beta.2 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Running 2.4.1 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Running 2.6.0-beta.2 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Running 2.6.0-beta.3 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Running 2.7.0-beta.0 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Running 2.7.0-beta.1 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Running 3.6.0-beta.1 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Number of clusters to migrate:  0
Running 5.0.0-beta.10 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Running 5.0.0-beta.13 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Running 5.1.2-latest.20210719.1 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
info: [STORE]: LOADED from C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
Running 5.0.0-alpha.0 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-hotbar-store.json
Running 5.0.0-alpha.2 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-hotbar-store.json
Running 5.0.0-beta.5 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-hotbar-store.json
Running 5.0.0-beta.10 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-hotbar-store.json
info: [STORE]: LOADED from C:\Users\<myuser>\AppData\Roaming\Lens\lens-hotbar-store.json
info: [STORE]: LOADED from C:\Users\<myuser>\AppData\Roaming\Lens\lens-extensions.json
info: [STORE]: LOADED from C:\Users\<myuser>\AppData\Roaming\Lens\lens-filesystem-provisioner-store.json
Running 5.0.0-alpha.0 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-weblink-store.json
Running 5.0.0-alpha.2 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-weblink-store.json
Running 5.0.0-beta.5 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-weblink-store.json
Running 5.0.0-beta.10 migration for C:\Users\<myuser>\AppData\Roaming\Lens\lens-weblink-store.json
info: [STORE]: LOADED from C:\Users\<myuser>\AppData\Roaming\Lens\lens-weblink-store.json
info: ­ƒöî Starting LensProxy
info: [LENS-PROXY]: Proxy server has started at 127.0.0.1:50035
info: ­ƒöÄ Testing LensProxy connection ...
info: ÔÜí LensProxy connection OK
debug: [EXTENSIONS-LOADER]: load on main
info: ­ƒûÑ´©Å  Starting WindowManager
info: ­ƒº® Initializing extensions
info: [EXTENSION-DISCOVERY] loading extensions from C:\Users\<myuser>\AppData\Roaming\Lens
debug: [EXTENSION-DISCOVERY]: 8 extensions loaded {"folderPath":"C:\\Users\\<myuser>\\AppData\\Local\\Programs\\Lens\\resources\\extensions","extensions":[{"id":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\kube-object-event-status\\package.json","absolutePath":"C:\\Users\\<myuser>\\AppData\\Local\\Programs\\Lens\\resources\\extensions\\kube-object-event-status\\kube-object-event-status-5.1.2-latest.20210719.1.tgz","manifestPath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\kube-object-event-status\\package.json","manifest":{"name":"kube-object-event-status","version":"5.1.2-latest.20210719.1","description":"Adds kube object status from events","renderer":"dist/renderer.js","lens":{"metadata":{},"styles":[]},"scripts":{"build":"webpack && npm pack","dev":"webpack --watch","test":"echo NO TESTS"},"files":["dist/**/*"],"dependencies":{},"devDependencies":{"@k8slens/extensions":"file:../../src/extensions/npm/extensions","ts-loader":"^8.0.4","typescript":"^4.3.2","webpack":"^4.44.2"}},"isBundled":true,"isEnabled":true,"isCompatible":true},{"id":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lenscloud-lens-extension\\package.json","absolutePath":"C:\\Users\\<myuser>\\AppData\\Local\\Programs\\Lens\\resources\\extensions\\lenscloud-lens-extension","manifestPath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lenscloud-lens-extension\\package.json","manifest":{"name":"lenscloud-lens-extension","publisher":"k8slens.dev","version":"5.1.2-latest.20210719.1","description":"Extension for getting access to Lens Spaces","homepage":"","repository":{"type":"git","url":"https://github.com/lensapp/lenscloud-lens-extension"},"bugs":{"url":"https://github.com/lensapp/lenscloud-lens-extension/issues"},"engines":{"lens":"^5.0.0-beta.11"},"main":"dist/main.js","renderer":"dist/renderer.js","scripts":{"start":"npm run start:dev","start:prod":"NODE_ENV=production npm run build-authenticator && NODE_ENV=production webpack --watch","start:dev":"NODE_ENV=development npm run build-authenticator && NODE_ENV=development webpack --watch","start:local":"NODE_ENV=development LENS_ENV=local npm run build-authenticator && NODE_ENV=development LENS_ENV=local webpack --watch","build-authenticator":"webpack --config webpack.authenticator.js && pkg ./build/lens-spaces-authenticator.js --public --no-bytecode --public-packages '*' -t node12 -o ./dist/lens-spaces-authenticator && (codesign --remove-signature ./dist/lens-spaces-authenticator || true)","build":"npm run clean && npm run webpack && npm run build-authenticator","webpack":"webpack","clean":"rm -rf ./dist && rm -rf ./build && rm *.tgz || true","lint":"eslint --ext ts,tsx --max-warnings=0 .","check:type":"tsc -p tsconfig.json --noEmit","test":"jest"},"files":["dist/**/*"],"devDependencies":{"@babel/preset-env":"^7.13.12","@babel/preset-react":"^7.13.13","@babel/preset-typescript":"^7.13.0","@emotion/css":"^11.1.3","@emotion/react":"^11.1.5","@emotion/styled":"^11.3.0","@k8slens/extensions":"file:../../src/extensions/npm/extensions","@material-ui/core":"^4.11.4","@material-ui/lab":"^4.0.0-alpha.57","@types/is-valid-domain":"^0.0.0","@types/jest":"^26.0.23","@types/js-yaml":"^4.0.1","@types/jsonwebtoken":"^8.5.0","@types/lodash":"^4.14.167","@types/node":"^12.19.13","@types/node-fetch":"^2.5.10","@types/randomcolor":"^0.5.5","@types/react":"^17.0.13","@types/react-table":"^7.0.28","@types/react-window":"^1.8.2","@types/uuid":"^8.3.1","@types/ws":"^7.4.4","@typescript-eslint/eslint-plugin":"^4.28.1","@typescript-eslint/parser":"^4.28.1","async-mutex":"^0.3.1","axios":"^0.21.1","bored-mplex":"^0.0.17","classnames":"^2.3.1","css-loader":"^5.2.6","date-fns":"^2.22.1","esbuild-loader":"^2.13.1","eslint":"^7.30.0","eslint-plugin-react":"^7.21.5","eslint-plugin-react-hooks":"^4.2.0","eslint-plugin-unused-imports":"^1.1.2","exponential-backoff":"^3.1.0","flex.box":"^3.4.4","got":"^11.8.2","http-proxy":"^1.18.1","initialism":"^1.0.4","is-valid-domain":"^0.0.20","jest":"^27.0.6","js-yaml":"^3.14.1","jsonwebtoken":"^8.5.1","jwt-decode":"^3.1.2","keycloak-js":"^13.0.1","lens-platform-sdk":"^5.0.1","mobx":"^6.3.0","mobx-react":"^7.1.0","net":"^1.0.2","node-fetch":"^2.6.1","node-machine-id":"^1.1.12","path":"^0.12.7","pkg":"^5.2.0","proxy-from-env":"^1.1.0","randomcolor":"^0.6.2","react":"^17.0.1","react-dom":"^17.0.1","react-table":"^7.6.3","react-window":"^1.8.6","sass":"^1.35.2","sass-loader":"^10.1.1","selfsigned":"^1.10.8","style-loader":"^2.0.0","ts-loader":"^8.0.14","tsconfig-paths-webpack-plugin":"^3.5.1","typed-emitter":"^1.3.1","typescript":"^4.3.5","url-loader":"^4.1.1","uuid":"^8.3.2","webpack":"^4.46.0","webpack-cli":"^3.3.11","ws":"^7.4.6"}},"isBundled":true,"isEnabled":true,"isCompatible":true},{"id":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-license\\package.json","absolutePath":"C:\\Users\\<myuser>\\AppData\\Local\\Programs\\Lens\\resources\\extensions\\license-menu-item\\lens-license-5.1.2-latest.20210719.1.tgz","manifestPath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-license\\package.json","manifest":{"name":"lens-license","version":"5.1.2-latest.20210719.1","description":"Lens IDE license menu item","main":"dist/main.js","scripts":{"build":"webpack -p && npm pack","dev":"webpack --watch","test":"jest --passWithNoTests --env=jsdom src $@","clean":"rm -rf *.tgz dist/"},"files":["dist/**/*"],"devDependencies":{"@k8slens/extensions":"file:../../src/extensions/npm/extensions","@types/node":"^14.14.14","@types/react":"^17.0.0","@types/webpack":"^4.41.17","conf":"^7.1.2","jest":"^26.6.3","mobx":"^6.3.2","react":"^16.13.1","ts-loader":"^8.0.4","ts-node":"^9.0.0","typescript":"^4.0.3","webpack":"^4.44.2","webpack-cli":"^3.3.11"}},"isBundled":true,"isEnabled":true,"isCompatible":true},{"id":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-metrics-cluster-feature\\package.json","absolutePath":"C:\\Users\\<myuser>\\AppData\\Local\\Programs\\Lens\\resources\\extensions\\metrics-cluster-feature\\lens-metrics-cluster-feature-5.1.2-latest.20210719.1.tgz","manifestPath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-metrics-cluster-feature\\package.json","manifest":{"name":"lens-metrics-cluster-feature","version":"5.1.2-latest.20210719.1","description":"Lens metrics cluster feature","renderer":"dist/renderer.js","lens":{"metadata":{},"styles":[]},"scripts":{"build":"webpack && npm pack","dev":"webpack --watch","test":"jest --passWithNoTests --env=jsdom src $@","clean":"rm -rf dist/ && rm *.tgz"},"files":["dist/**/*","resources/**/*"],"devDependencies":{"@k8slens/extensions":"file:../../src/extensions/npm/extensions","jest":"^26.6.3","semver":"^7.3.2","ts-loader":"^8.0.4","typescript":"^4.3.2","webpack":"^4.44.2"}},"isBundled":true,"isEnabled":true,"isCompatible":true},{"id":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-node-menu\\package.json","absolutePath":"C:\\Users\\<myuser>\\AppData\\Local\\Programs\\Lens\\resources\\extensions\\node-menu\\lens-node-menu-5.1.2-latest.20210719.1.tgz","manifestPath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-node-menu\\package.json","manifest":{"name":"lens-node-menu","version":"5.1.2-latest.20210719.1","description":"Lens node menu","renderer":"dist/renderer.js","lens":{"metadata":{},"styles":[]},"scripts":{"build":"webpack && npm pack","dev":"webpack --watch","test":"jest --passWithNoTests --env=jsdom src $@"},"files":["dist/**/*"],"dependencies":{},"devDependencies":{"@k8slens/extensions":"file:../../src/extensions/npm/extensions","jest":"^26.6.3","ts-loader":"^8.0.4","typescript":"^4.3.2","webpack":"^4.44.2"}},"isBundled":true,"isEnabled":true,"isCompatible":true},{"id":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-pod-menu\\package.json","absolutePath":"C:\\Users\\<myuser>\\AppData\\Local\\Programs\\Lens\\resources\\extensions\\pod-menu\\lens-pod-menu-5.1.2-latest.20210719.1.tgz","manifestPath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-pod-menu\\package.json","manifest":{"name":"lens-pod-menu","version":"5.1.2-latest.20210719.1","description":"Lens pod menu","renderer":"dist/renderer.js","lens":{"metadata":{},"styles":[]},"scripts":{"build":"webpack && npm pack","dev":"webpack --watch","test":"jest --passWithNoTests --env=jsdom src $@"},"files":["dist/**/*"],"dependencies":{},"devDependencies":{"@k8slens/extensions":"file:../../src/extensions/npm/extensions","jest":"^26.6.3","ts-loader":"^8.0.4","typescript":"^4.3.2","webpack":"^4.44.2"}},"isBundled":true,"isEnabled":true,"isCompatible":true},{"id":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-survey\\package.json","absolutePath":"C:\\Users\\<myuser>\\AppData\\Local\\Programs\\Lens\\resources\\extensions\\survey\\lens-survey-5.1.2-latest.20210719.1.tgz","manifestPath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-survey\\package.json","manifest":{"name":"lens-survey","version":"5.1.2-latest.20210719.1","description":"Lens IDE survey","main":"dist/main.js","renderer":"dist/renderer.js","lens":{"metadata":{},"styles":[]},"scripts":{"build":"webpack -p && npm pack","dev":"webpack --watch","test":"jest --passWithNoTests --env=jsdom src $@"},"files":["dist/**/*"],"devDependencies":{"@k8slens/extensions":"file:../../src/extensions/npm/extensions","@types/node":"^14.14.14","@types/react":"^17.0.0","@types/webpack":"^4.41.17","conf":"^7.1.2","got":"^11.8.1","jest":"^26.6.3","mobx":"^6.3.2","mobx-react":"^7.2.0","node-machine-id":"^1.1.12","react":"^16.13.1","refiner-js":"^1.0.1","ts-loader":"^8.0.4","typescript":"^4.0.3","webpack":"^4.44.2","webpack-cli":"^3.3.11"}},"isBundled":true,"isEnabled":true,"isCompatible":true},{"id":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-telemetry\\package.json","absolutePath":"C:\\Users\\<myuser>\\AppData\\Local\\Programs\\Lens\\resources\\extensions\\telemetry\\lens-telemetry-5.1.2-latest.20210719.1.tgz","manifestPath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\lens-telemetry\\package.json","manifest":{"name":"lens-telemetry","version":"5.1.2-latest.20210719.1","description":"Lens IDE telemetry","main":"dist/main.js","renderer":"dist/renderer.js","lens":{"metadata":{},"styles":[]},"scripts":{"build":"webpack -p && npm pack","dev":"webpack --watch","test":"jest --passWithNoTests --env=jsdom src $@"},"files":["dist/**/*"],"devDependencies":{"@k8slens/extensions":"file:../../src/extensions/npm/extensions","@types/analytics-node":"^3.1.3","@types/node":"^14.14.14","@types/react":"^17.0.0","@types/universal-analytics":"^0.4.4","analytics-node":"^3.4.0-beta.3","conf":"^7.1.2","jest":"^26.6.3","mobx":"^6.3.2","mobx-react":"^7.1.0","node-machine-id":"^1.1.12","react":"^16.13.1","ts-loader":"^8.0.4","typescript":"^4.0.3","universal-analytics":"^0.4.23","webpack":"^4.44.2","webpack-cli":"^3.3.11"}},"isBundled":true,"isEnabled":true,"isCompatible":true}]}
info: [EXTENSION-INSTALLER] installing dependencies at C:\Users\<myuser>\AppData\Roaming\Lens
info: [WINDOW-MANAGER]:┬áLoading Main window from url: http://localhost:50035 ...
info: [EXTENSION-INSTALLER] dependencies installed at C:\Users\<myuser>\AppData\Roaming\Lens
debug: [EXTENSION-DISCOVERY]: 1 extensions loaded {"folderPath":"C:\\Users\\<myuser>\\.k8slens\\extensions","extensions":[{"id":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\@nevalla\\kube-resource-map\\package.json","absolutePath":"C:\\Users\\<myuser>\\.k8slens\\extensions\\nevalla--kube-resource-map","manifestPath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\node_modules\\@nevalla\\kube-resource-map\\package.json","manifest":{"name":"@nevalla/kube-resource-map","version":"0.2.0","description":"Visualization of Kubernetes resources and their relations","publisher":"Lauri Nevala","renderer":"dist/renderer.js","engines":{"node":">=12.0 <13.0"},"keywords":["lens","extension","k8slens","kubernetes"],"files":["dist/"],"scripts":{"start":"webpack --watch","build":"npm run clean && webpack","clean":"rm -rf ./dist","test":"jest"},"jest":{"runner":"@jest-runner/electron","testEnvironment":"@jest-runner/electron/environment"},"contributes":{"crds":[],"cloudProviders":[],"kubernetesDistros":[]},"dependencies":{"react-dom":"^17.0.1"},"devDependencies":{"@jest-runner/electron":"^3.0.0","@k8slens/extensions":"4.0.8","@testing-library/jest-dom":"^5.11.6","@testing-library/react":"^11.2.2","@types/d3-force":"^2.1.0","@types/jest":"^26.0.15","@types/node":"^12.12.9","@types/react":"^17.0.0","@typescript-eslint/eslint-plugin":"^4.8.1","@typescript-eslint/parser":"^4.8.1","css-loader":"^5.0.1","d3-force":"^2.1.1","electron":"^11.0.3","jest":"^26.6.3","mobx":"^5.15.7","mobx-react":"^6.2.2","node-sass":"^5.0.0","react":"^17.0.1","react-force-graph":"1.37.5","sass-loader":"^10.1.0","style-loader":"^2.0.0","ts-loader":"^8.0.11","typescript":"^4.1.2","webpack":"^4.44.2","webpack-cli":"^3.3.11"}},"isBundled":false,"isEnabled":false,"isCompatible":false}]}
info: [EXTENSION-INSTALLER] installing package from C:\Users\<myuser>\.k8slens\extensions\nevalla--kube-resource-map to C:\Users\<myuser>\AppData\Roaming\Lens
info: [KUBECONFIG-SYNC]: starting requested syncs
info: ­ƒôí Checking for app updates
info: Checking for update
info: [KUBECONFIG-SYNC]: starting sync of file/folder {"filePath":"C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\kubeconfigs"}
debug: [KUBECONFIG-SYNC]: 1 files/folders watched {"files":["C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\kubeconfigs"]}
info: [KUBECONFIG-SYNC]: starting sync of file/folder {"filePath":"C:\\Users\\<myuser>\\.kube"}
debug: [KUBECONFIG-SYNC]: 2 files/folders watched {"files":["C:\\Users\\<myuser>\\AppData\\Roaming\\Lens\\kubeconfigs","C:\\Users\\<myuser>\\.kube"]}
info: Generated new staging user ID: e4e3205c-eaf0-571d-adb1-735d45aa6ad1
debug: [KUBECONFIG-SYNC]: file changed {"filePath":"C:\\Users\\<myuser>\\.kube\\config"}
debug: [KUBECONFIG-SYNC]: file changed {"filePath":"C:\\Users\\<myuser>\\.kube\\config.old"}
debug: [KUBECONFIG-SYNC]: context failed validation: Error: No valid cluster object provided in kubeconfig for context '****' {"context":"****","filePath":"C:\\Users\\<myuser>\\.kube\\config"}
debug: [KUBECONFIG-SYNC]: File now has 7 entries {"filePath":"C:\\Users\\<myuser>\\.kube\\config"}
debug: [CLUSTER]: Cluster init success {"id":"df39fcab76ce45951ae57f0f9078fb33","context":"****","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config","contextName":"****"}
debug: [CLUSTER]: Cluster init success {"id":"881ecad79fa3a7451e36597c5f44ed26","context":"eos-dev-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config","contextName":"eos-dev-aks"}
debug: [CLUSTER]: Cluster init success {"id":"6cb01d8995ddeea9572dee41a35b3f29","context":"eos-dev-aks-admin","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config","contextName":"eos-dev-aks-admin"}
debug: [CLUSTER]: Cluster init success {"id":"54f0b3dc47cc49318260620316fa3bfc","context":"eos-dev2-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config","contextName":"eos-dev2-aks"}
debug: [CLUSTER]: Cluster init success {"id":"3a5d6fdec3d91962eb2d0c21a35d6b0c","context":"eos-prd-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config","contextName":"eos-prd-aks"}
debug: [CLUSTER]: Cluster init success {"id":"3b03cfd52150f2dde9070f09f2045dc5","context":"eos-ref-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config","contextName":"eos-ref-aks"}
debug: [CLUSTER]: Cluster init success {"id":"f18bd8e78e33f8a1bc1ca11104621d00","context":"eos-tst-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config","contextName":"eos-tst-aks"}
debug: [CLUSTER]: Cluster init success {"id":"df39fcab76ce45951ae57f0f9078fb33","context":"****","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"881ecad79fa3a7451e36597c5f44ed26","context":"eos-dev-aks","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"6cb01d8995ddeea9572dee41a35b3f29","context":"eos-dev-aks-admin","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"54f0b3dc47cc49318260620316fa3bfc","context":"eos-dev2-aks","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"3a5d6fdec3d91962eb2d0c21a35d6b0c","context":"eos-prd-aks","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"3b03cfd52150f2dde9070f09f2045dc5","context":"eos-ref-aks","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"f18bd8e78e33f8a1bc1ca11104621d00","context":"eos-tst-aks","apiUrl":"****"}
info: [STORE]: SAVING C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
debug: [KUBECONFIG-SYNC]: context failed validation: Error: No valid cluster object provided in kubeconfig for context '****' {"context":"****","filePath":"C:\\Users\\<myuser>\\.kube\\config.old"}
debug: [KUBECONFIG-SYNC]: File now has 6 entries {"filePath":"C:\\Users\\<myuser>\\.kube\\config.old"}
debug: [CLUSTER]: Cluster init success {"id":"6727bc6b5cae7ae481507e8c5f29bf77","context":"****","****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config.old","****"}
debug: [CLUSTER]: Cluster init success {"id":"687da07a338d09f02b3c488dd464cca3","context":"eos-dev-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config.old","contextName":"eos-dev-aks"}
debug: [CLUSTER]: Cluster init success {"id":"f247f4b572ccb0133d5d0e1d6b7f7a58","context":"eos-dev2-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config.old","contextName":"eos-dev2-aks"}
debug: [CLUSTER]: Cluster init success {"id":"f914bf94b6611707076edcf97a144f31","context":"eos-prd-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config.old","contextName":"eos-prd-aks"}
debug: [CLUSTER]: Cluster init success {"id":"79a86caf71110e86115002d59f97940e","context":"eos-ref-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config.old","contextName":"eos-ref-aks"}
debug: [CLUSTER]: Cluster init success {"id":"fd57bda030c8f00a598cc4be4e589911","context":"eos-tst-aks","apiUrl":"****"}
debug: [KUBECONFIG-SYNC]: Added new cluster from sync {"filePath":"C:\\Users\\<myuser>\\.kube\\config.old","contextName":"eos-tst-aks"}
debug: [CLUSTER]: Cluster init success {"id":"6727bc6b5cae7ae481507e8c5f29bf77","context":"****","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"687da07a338d09f02b3c488dd464cca3","context":"eos-dev-aks","apiUrl":"****"}
Error: npm ERR! code EAI_AGAIN
npm ERR! errno EAI_AGAIN
npm ERR! request to https://registry.npmjs.org/react-dom failed, reason: getaddrinfo EAI_AGAIN registry.npmjs.org

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\<myuser>\npm-cache\_logs\2021-07-22T09_32_03_737Z-debug.log

    at ChildProcess.<anonymous> (C:\Users\<myuser>\AppData\Local\Programs\Lens\resources\app.asar\static\build\main.js:1:205192)
    at ChildProcess.emit (events.js:223:5)
    at ChildProcess.EventEmitter.emit (domain.js:475:20)
    at maybeClose (internal/child_process.js:1021:16)
    at Socket.<anonymous> (internal/child_process.js:430:11)
    at Socket.emit (events.js:223:5)
    at Socket.EventEmitter.emit (domain.js:475:20)
    at Pipe.<anonymous> (net.js:664:12)
Trace
    at App.<anonymous> (C:\Users\<myuser>\AppData\Local\Programs\Lens\resources\app.asar\static\build\main.js:1:257521)
debug: [CLUSTER]: Cluster init success {"id":"f247f4b572ccb0133d5d0e1d6b7f7a58","context":"eos-dev2-aks","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"f914bf94b6611707076edcf97a144f31","context":"eos-prd-aks","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"79a86caf71110e86115002d59f97940e","context":"eos-ref-aks","apiUrl":"****"}
debug: [CLUSTER]: Cluster init success {"id":"fd57bda030c8f00a598cc4be4e589911","context":"eos-tst-aks","apiUrl":"****"}
info: [STORE]: SAVING C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
info: [WINDOW-MANAGER]: Main window loaded
info: [WINDOW-MANAGER]: Main window has reported that it has loaded
info: Update for version 5.1.2-latest.20210719.1 is not available (latest version: 5.1.2-latest.20210719.1, downgrade is disallowed).
info: [MENU]: navigating to /preferences
info: [WINDOW-MANAGER]: Main window has reported that it has loaded
debug: UserStore.fromStore() {"lastSeenAppVersion":"0.0.0","preferences":{"httpsProxy":"<proxy>:8080"}}
info: [STORE]: SAVING C:\Users\<myuser>\AppData\Roaming\Lens\lens-user-store.json
info: [STORE]: SAVING C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
info: [CLUSTER]: activate {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":true}
info: [CLUSTER]: bind events {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":true}
info: [CLUSTER]: reconnect {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":true}
debug: Set kubectl version 1.18.15 for cluster version 1.18.15 using version map
debug: Acquired a lock for 1.18.15
debug: Releasing lock for 1.18.15
debug: spawning kubectl proxy with args: proxy,-p,0,--kubeconfig,C:\Users\<myuser>\.kube\config,--context,eos-dev-aks,--accept-hosts,****,--reject-paths,^[^/]
info: [KUBE-AUTH]: out-channel "kube-auth:881ecad79fa3a7451e36597c5f44ed26" {"data":"Authentication proxy started\n","meta":{"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":true}}
error: [LENS-PROXY]: http proxy errored for cluster: Error: connect ECONNREFUSED 127.0.0.1:80 {"url":"/apis/networking.k8s.io/v1"}
debug: Failed proxy to target: {
  "protocol": "http:",
  "slashes": true,
  "auth": null,
  "host": "127.0.0.1",
  "port": null,
  "hostname": "127.0.0.1",
  "hash": null,
  "search": null,
  "query": null,
  "pathname": "/:undefined",
  "path": "/:undefined",
  "href": "http://127.0.0.1/:undefined"
}
error: [LENS-PROXY]: http proxy errored for cluster: Error: connect ECONNREFUSED 127.0.0.1:80 {"url":"/apis/apiextensions.k8s.io"}
debug: Failed proxy to target: {
  "protocol": "http:",
  "slashes": true,
  "auth": null,
  "host": "127.0.0.1",
  "port": null,
  "hostname": "127.0.0.1",
  "hash": null,
  "search": null,
  "query": null,
  "pathname": "/:undefined",
  "path": "/:undefined",
  "href": "http://127.0.0.1/:undefined"
}
error: [LENS-PROXY]: http proxy errored for cluster: Error: connect ECONNREFUSED 127.0.0.1:80 {"url":"/version"}
debug: Failed proxy to target: {
  "protocol": "http:",
  "slashes": true,
  "auth": null,
  "host": "127.0.0.1",
  "port": null,
  "hostname": "127.0.0.1",
  "hash": null,
  "search": null,
  "query": null,
  "pathname": "/:undefined",
  "path": "/:undefined",
  "href": "http://127.0.0.1/:undefined"
}
error: Failed to connect cluster "eos-dev-aks": StatusCodeError: 500 - "Oops, something went wrong.\nError: connect ECONNREFUSED 127.0.0.1:80"
info: [CLUSTER]: activate {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":false}
info: [CLUSTER]: reconnect {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":false}
debug: [KUBE-AUTH]: stopping local proxy {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":false}
error: [LENS-PROXY]: http proxy errored for cluster: Error: connect ECONNREFUSED 127.0.0.1:80 {"url":"/apis/extensions/v1beta1"}
debug: Failed proxy to target: {
  "protocol": "http:",
  "slashes": true,
  "auth": null,
  "host": "127.0.0.1",
  "port": null,
  "hostname": "127.0.0.1",
  "hash": null,
  "search": null,
  "query": null,
  "pathname": "/:undefined",
  "path": "/:undefined",
  "href": "http://127.0.0.1/:undefined"
}
debug: Acquired a lock for 1.18.15
debug: Releasing lock for 1.18.15
debug: spawning kubectl proxy with args: proxy,-p,0,--kubeconfig,C:\Users\<myuser>\.kube\config,--context,eos-dev-aks,--accept-hosts,****,--reject-paths,^[^/]
info: [KUBE-AUTH]: out-channel "kube-auth:881ecad79fa3a7451e36597c5f44ed26" {"data":"Authentication proxy started\n","meta":{"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":false}}
error: [LENS-PROXY]: http proxy errored for cluster: Error: connect ECONNREFUSED 127.0.0.1:80 {"url":"/apis/networking.k8s.io"}
debug: Failed proxy to target: {
  "protocol": "http:",
  "slashes": true,
  "auth": null,
  "host": "127.0.0.1",
  "port": null,
  "hostname": "127.0.0.1",
  "hash": null,
  "search": null,
  "query": null,
  "pathname": "/:undefined",
  "path": "/:undefined",
  "href": "http://127.0.0.1/:undefined"
}
error: [LENS-PROXY]: http proxy errored for cluster: Error: connect ECONNREFUSED 127.0.0.1:80 {"url":"/version"}
debug: Failed proxy to target: {
  "protocol": "http:",
  "slashes": true,
  "auth": null,
  "host": "127.0.0.1",
  "port": null,
  "hostname": "127.0.0.1",
  "hash": null,
  "search": null,
  "query": null,
  "pathname": "/:undefined",
  "path": "/:undefined",
  "href": "http://127.0.0.1/:undefined"
}
error: Failed to connect cluster "eos-dev-aks": StatusCodeError: 500 - "Oops, something went wrong.\nError: connect ECONNREFUSED 127.0.0.1:80"
info: [CLUSTER]: activate {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":false}
info: [CLUSTER]: reconnect {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":false}
debug: [KUBE-AUTH]: stopping local proxy {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":false}
debug: Acquired a lock for 1.18.15
debug: Releasing lock for 1.18.15
debug: spawning kubectl proxy with args: proxy,-p,0,--kubeconfig,C:\Users\<myuser>\.kube\config,--context,eos-dev-aks,--accept-hosts,****,--reject-paths,^[^/]
info: [KUBE-AUTH]: out-channel "kube-auth:881ecad79fa3a7451e36597c5f44ed26" {"data":"Authentication proxy started\n","meta":{"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":false,"online":false,"accessible":false,"disconnected":false}}
info: [STORE]: SAVING C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
debug: Dumping KubeConfig: {"apiVersion":"v1","kind":"Config","preferences":{},"current-context":"eos-dev-aks","clusters":[{"name":"eos-dev-aks","cluster":{"server":"http://127.0.0.1:50035/881ecad79fa3a7451e36597c5f44ed26"}}],"contexts":[{"name":"eos-dev-aks","context":{"cluster":"eos-dev-aks","user":"proxy","namespace":"grafana"}}],"users":[{"name":"proxy","user":{}}]}
debug: Created temp kubeconfig "eos-dev-aks" at "C:\Users\<myuser>\AppData\Local\Temp\kubeconfig-881ecad79fa3a7451e36597c5f44ed26":
apiVersion: v1
kind: Config
preferences: {}
current-context: eos-dev-aks
clusters:
  - name: eos-dev-aks
    cluster:
      server: 'http://127.0.0.1:50035/881ecad79fa3a7451e36597c5f44ed26'
contexts:
  - name: eos-dev-aks
    context:
      cluster: eos-dev-aks
      user: proxy
      namespace: grafana
users:
  - name: proxy
    user: {}

debug: Set kubectl version 1.19.7 for cluster version v1.19.9 using version map
debug: Acquired a lock for 1.19.7
info: Downloading kubectl 1.19.7 from https://storage.googleapis.com/kubernetes-release/release/v1.19.7/bin/windows/amd64/kubectl.exe to C:\Users\<myuser>\AppData\Roaming\Lens\binaries\kubectl\1.19.7\kubectl.exe
error: tunneling socket could not be established, cause=getaddrinfo EAI_AGAIN 8080 {"code":"ECONNRESET"}
error: tunneling socket could not be established, cause=getaddrinfo EAI_AGAIN 8080 {"code":"ECONNRESET"}
debug: Releasing lock for 1.19.7
debug: Releasing lock for 1.19.7
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
Error: Lock is already released
    at C:\Users\<myuser>\AppData\Local\Programs\Lens\resources\app.asar\node_modules\proper-lockfile\lib\lockfile.js:257:60
    at C:\Users\<myuser>\AppData\Local\Programs\Lens\resources\app.asar\node_modules\proper-lockfile\lib\adapter.js:39:9
    at new Promise (<anonymous>)
    at C:\Users\<myuser>\AppData\Local\Programs\Lens\resources\app.asar\node_modules\proper-lockfile\lib\adapter.js:30:25
    at C:\Users\<myuser>\AppData\Local\Programs\Lens\resources\app.asar\static\build\main.js:1:84666
    at async sa.ensureKubectl (C:\Users\<myuser>\AppData\Local\Programs\Lens\resources\app.asar\static\build\main.js:1:133314)
warn: PrometheusLens: failed to list services: services "prometheus" not found
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
warn: PrometheusLens: failed to list services: services "prometheus" not found
info: [STORE]: SAVING C:\Users\<myuser>\AppData\Roaming\Lens\lens-cluster-store.json
warn: PrometheusLens: failed to list services: services "prometheus" not found
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
warn: PrometheusLens: failed to list services: services "prometheus" not found
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
warn: PrometheusLens: failed to list services: services "prometheus" not found
warn: PrometheusLens: failed to list services: services "prometheus" not found
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
warn: PrometheusLens: failed to list services: services "prometheus-server" not found
info: [CLUSTER]: refresh {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":true,"online":true,"accessible":true,"disconnected":false}
info: [CLUSTER]: refresh {"id":"881ecad79fa3a7451e36597c5f44ed26","name":"eos-dev-aks","ready":true,"online":true,"accessible":true,"disconnected":false}

```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
your kubeconfig here
```

**Additional context**
Please also see https://github.com/lensapp/lens/issues/1690#issuecomment-884506754 where I was asked to open this new ticket

--- a/src/extensions/extension-discovery.ts
+++ b/src/extensions/extension-discovery.ts
@@ -20,7 +20,7 @@
  */
 
 import { watch } from "chokidar";
-import { ipcRenderer } from "electron";
+import { dialog, ipcRenderer } from "electron";
 import { EventEmitter } from "events";
 import fse from "fs-extra";
 import { observable, reaction, when, makeObservable } from "mobx";
@@ -393,7 +393,14 @@ export class ExtensionDiscovery extends Singleton {
 
     for (const extension of userExtensions) {
       if ((await fse.pathExists(extension.manifestPath)) === false) {
-        await this.installPackage(extension.absolutePath);
+        try {
+          await this.installPackage(extension.absolutePath);
+        } catch (error) {
+          const message = error.message || error?.toString() || "unknown error";
+          const { name, version } = extension.manifest;
+
+          dialog.showErrorBox("Lens Extension Install Failure", `Failed to install user extension ${name}@${version}: ${message}`);
+        }
       }
     }
 

--- a/src/extensions/extension-discovery.ts
+++ b/src/extensions/extension-discovery.ts
@@ -20,7 +20,7 @@
  */
 
 import { watch } from "chokidar";
-import { dialog, ipcRenderer } from "electron";
+import { ipcRenderer } from "electron";
 import { EventEmitter } from "events";
 import fse from "fs-extra";
 import { observable, reaction, when, makeObservable } from "mobx";
@@ -396,10 +396,10 @@ export class ExtensionDiscovery extends Singleton {
         try {
           await this.installPackage(extension.absolutePath);
         } catch (error) {
-          const message = error.message || error?.toString() || "unknown error";
+          const message = error.message || error || "unknown error";
           const { name, version } = extension.manifest;
 
-          dialog.showErrorBox("Lens Extension Install Failure", `Failed to install user extension ${name}@${version}: ${message}`);
+          logger.error(`${logModule}: failed to install user extension ${name}@${version}`, message);
         }
       }
     }

